nft_ingester/src/main.rs

```diff
@@ -77,7 +93,7 @@ async fn main() {
         .await,
     );
     // Start up backfiller process.
-    tasks.push(backfiller(pool.clone()).await);
+    tasks.push(backfiller::<RedisMessenger>(pool.clone(), config.clone()).await);
```

Comment:
Yes always running, but asleep unless a bubblegum instruction is parsed.  Then it will just check just the tree for which a new row was added to the `backfill_items` table, typically it will see no gap and then do a quick delete of the extra row and mark the new row as backfilled.